iOS/Flows/Conversation/Threads/ConversationThreadViewController.swift

```diff
@@ -107,7 +106,11 @@ class ConversationThreadViewController: DiffableCollectionViewController<Convers
             let messages = Array(self.messageController.replies.asConversationCollectionItems)
 
             if let channelId = self.message.cid {
-                data[.conversation(channelId)] = messages
+                data[.conversation(channelId)] = []
+                data[.conversation(channelId)]?.append(contentsOf: messages)
+                if !self.messageController.hasLoadedAllPreviousReplies {
+                    data[.conversation(channelId)]?.append(contentsOf: [.loadMore])
+                }
```

Comment:
Adding the load more cell consistently at the END. This makes indexing cells A LOT less annoying because off-by-one errors are more rare.